builtin/providers/namecheap/resource_namecheap_record.go

```diff
@@ -0,0 +1,149 @@
+package namecheap
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+
+	"github.com/HX-Rd/namecheap"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+var mutex = &sync.Mutex{}
```

Comment:
Is there something about the remote API that makes it necessary to take a mutex here?